src/com/facebook/buck/intellij/ideabuck/src/com/facebook/buck/intellij/ideabuck/actions/BuckDebugSelectedTestFromProjectAction.java

```diff
@@ -0,0 +1,26 @@
+package com.facebook.buck.intellij.ideabuck.actions;
+
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+
+/*
```

Comment:
Can we make the copyright consistent here? It seems to be always at the top of the file elsewhere, but not in this file